Kernel/Memory/Region.cpp

```diff
@@ -496,6 +496,7 @@ PageFaultResponse Region::handle_zero_fault(size_t page_index_in_region, Physica
 {
     VERIFY(vmobject().is_anonymous());
 
+    auto& anon_vmobject = static_cast<AnonymousVMObject&>(vmobject());
```

Comment:
nit: `anonymous_vmobject` would be more in line with serenityos naming (https://github.com/SerenityOS/serenity/blob/master/Documentation/CodingStyle.md "Use full words, except in the rare case where an abbreviation would be more canonical and easier to understand.")

Reply:
Ok, I replaced `anon_vmobject` with `anonymous_vmobject`